packages/@ember/-internals/glimmer/lib/helpers/action.ts

```diff
@@ -415,6 +415,10 @@ function makeClosureAction(
       {
         until: '3.25.0',
         id: 'actions.custom-invoke-invokable',
+        for: 'ember-source',
+        since: {
+          enabled: '3.21.0',
```

Comment:
not sure if this is different from using `{{action}}` has a helper in templates?

Reply:
Ya, it is different, but it was never exposed as public API. This is a private API deprecation specifically added so that we don't break a private API usages within ember-concurrency.

Reply:
Should be 3.23.0-beta.1